website/content/docs/commands/validate.mdx

```diff
@@ -33,12 +33,16 @@ Errors validating build 'vmware'. 1 error(s) occurred:
   configuration is not validated.
 
 - `-except=foo,bar,baz` - Validates all the builds except those with the
-  comma-separated names. Build names by default are the names of their
-  builders, unless a specific `name` attribute is specified within the configuration.
+  comma-separated names. Name is a required block label in HCL, but in legacy
```

Comment:
I understood the issue differently. 
They said 

> stopped running after I added a name field inside each build source block to differentiate them:

For HCL2, I thought they needed more information about when the `name` is set it requires to be used instead of the one giving by the `source` block definition.

Reply:
hmm that makes sense but I'm struggling to come up with a concise description. Perhaps the best solution is to provide a toy example that demonstrates the behavior.

Reply:
That's a good idea! I did that in the `override` docs in the PR I have it open because I get often confused with it as well. In that case, the override won't work if not using the given value of the `name` attribute. The same way as only/except 🤔

Reply:
turns out we already have one https://www.packer.io/docs/templates/hcl_templates/onlyexcept